### PR TITLE
Add model class ReducedWongWangExcIOInhI

### PR DESCRIPTION
**Pull request by liadomide**
@dionperd created a new model class: ReducedWongWangExcIOInhI 
We would like to have that available for HBP collab really soon

### CODE REVIEW

File: tvb/simulator/models/__init__.py [hunk 1]
```diff
@@ -46,8 +46,10 @@
...
+from .wong_wang_exc_io_inh_i import ReducedWongWangExcIOInhI
 from .linear import Linear
 from .hopfield import Hopfield
-from .epileptor import Epileptor
+from .epileptor import Epileptor, Epileptor2D
+from .JCepileptor import JC_Epileptor
 from .epileptorcodim3 import EpileptorCodim3, EpileptorCodim3SlowMod
-from .Zerlaut import Zerlaut_adaptation_first_order, Zerlaut_adaptation_second_order
\ No newline at end of file
+from .Zerlaut import Zerlaut_adaptation_first_order, Zerlaut_adaptation_second_order
```
Comment (maedoc) [inline tvb/simulator/models/__init__.py:55]: I see a few extraneous imports here, should you rebase this against the latest trunk? if the Zerlaut models are not in trunk, the import lines fail.  If they are, you shouldn't need this change

Comment (liadomide) [reply tvb/simulator/models/__init__.py:55]: Zerlaut models are in trunk already.
The last line is not really changed with my current commit, only newline was added automatically by my IDE.
But maybe I do not understand completely your comment....

Comment (maedoc) [reply tvb/simulator/models/__init__.py:55]: I see what you mean.  it's not a problem, I just didn't expect to see extra diff lines that aren't for the model being added.

Comment (liadomide) [reply tvb/simulator/models/__init__.py:55]: True.... 
I also did a review of this init file, in addition to adding the current model, and did not reflect that in the commit comment.

**Comment by liadomide** [comment]
Original code shared by @dionperd can be inspected here: 
https://github.com/dionperd/tvb-multiscale/blob/master/tvb_nest/tvb_models/tvb_reduced_wong_wang_exc_io_inh_i.py
https://github.com/dionperd/tvb-multiscale/blob/master/tvb_nest/tests/test_reduced_wong_wang_exc_io_inh_i.py

**Comment by maedoc** [comment]
ok thanks.  I wasn't aware Denis was working on Nest TVB interfacing.  I intended to implement some functionality in that direction as well, within TVB, but perhaps I should wait until some coordination can happen.

**Comment by dionperd** [comment]
Check it out here:
https://github.com/dionperd/tvb-multiscale

**Comment by liadomide** [comment]
Is there any objection to merging this?
An ipython notebook example will follow soon, but we would benefit if this merge goes soon in trunk, so that we can have a Pypi release while working on the notebook example.

File: tvb/simulator/models/wong_wang_exc_io_inh_i.py [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+
+        """
+        S = state_variables[:, :]
+        S[S < 0] = 0.
+        S[S > 1] = 1.
+
+        c_0 = coupling[0, :]
+
+        # if applicable
+        lc_0 = local_coupling * S[0]
```
Comment (maedoc) [inline tvb/simulator/models/wong_wang_exc_io_inh_i.py:275]: @dionperd this local coupling is between excitatory populations, which may be not be correct. Please see my comment [here](https://github.com/the-virtual-brain/tvb-library/pull/86#issuecomment-465068962) on Julie's PR.

**Pull request merged**